notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb

```diff
@@ -346,26 +346,6 @@
         "! gsutil mb -l $REGION -p $PROJECT_ID $BUCKET_URI"
```

Comment:
<p>I wonder if this batch of code should be closer to the actual invocation of the job.  It's a little hard to remember what you set all the parameters to up here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>